Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[required]
+python_version = "3.7"
```

Comment:
I also changed this section to:

    [requires]
    python_version = "3"

and that seemed to work fine in general. I think this should have been `[requires]` instead of `[required]` in the first place from my quick [look at the docs](http://pipenv.readthedocs.io/en/latest/basics/#specifying-versions-of-python).

Reply:
Changed. Looks like fixing this got the python version into the Pipfile.lock.